XamarinCommunityToolkit/Views/ISideMenuList.shared.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.Generic;
+using Xamarin.Forms;
+
+namespace Microsoft.Toolkit.Xamarin.Forms.UI.Views
+{
+	public interface ISideMenuList<T> : IList<T> where T : View
```

Comment:
Should there be helper methods for AddLeft and AddRight? Which have the second parameter `SideMenuPosition` already filled? I can imagine people using mostly one of both and not want to specify that parameter each time

Reply:
I guess most of the people will work through the xaml. So they will use attached property, but I am OK with adding such methods. They will be a good addition